proto/config.proto

```diff
@@ -433,11 +433,121 @@ message PSPassthroughOptions
 
 message WiiOptions
 {
+    message AnalogAxis
+    {
+        optional int32 axisType = 1;
+        optional int32 minRange = 2;
+        optional int32 maxRange = 3;
+    }
+
+    message StickOptions
+    {
+        optional AnalogAxis x = 1;
+        optional AnalogAxis y = 2;
+    }
+    
+	message NunchukOptions
+	{
+		optional int32 buttonC = 1;
```

Comment:
for some reason, some of the indents are tabs and some are spaces

Reply:
Will be fixed in next push.